packages/core/jest.config.js

```diff
@@ -6,4 +6,5 @@ module.exports = {
   },
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
   coverageDirectory: '../../coverage/packages/core',
+  globals: { window: {} },
```

Comment:
Reason for adding jest globals:
without it there's error
<img width="418" alt="image" src="https://user-images.githubusercontent.com/28751745/165328628-722d9d0a-2758-4af2-9bc8-952e1173f016.png">
And I think it's better to keep jest running in 'node' env

Reply:
Just use a `typeof window` check first 😉 As we will get the same error with SSR.

```
export const isDevToolsEnabled: boolean = typeof window !== 'undefined' && window?.sessionStorage?.getItem('__GRIFFEL_DEVTOOLS__');
```

Reply:
Ahh nice :)